backend/src/v5/models/users.js

```diff
@@ -158,4 +162,53 @@ User.deleteFavourites = async (username, teamspace, favouritesToRemove) => {
 	}
 };
 
+const changePassword = async (username, newPassword) => {
+	const updateUserCmd = {
+		updateUser: username,
+		pwd: newPassword,
+	};
+
+	await db.runCommand('admin', updateUserCmd);
+	await updateUser(username, { $unset: { 'customData.resetPasswordToken': 1 } });
+};
+
+User.updateProfile = async (username, updatedProfile) => {
```

Comment:
can we move this to the processor?

Basically models should be tightly knitted to the schema of the database, this is mostly trying to convert data format from what we get from network to what we need to do in the datbaase, which is more of a proccessor's  job

So the processor function should just call a `changePassword` function, and/or `update` function in here.